cluster/calcium/resource.go

```diff
@@ -4,6 +4,8 @@ import (
 	"context"
 	"sort"
 
+	"github.com/sanity-io/litter"
+
```

Comment:
Maybe we can remove this empty line, it breaks the style.

Reply:
format by go format ... no idea why